doc/ref_model/chapters/chapter06.md

```diff
@@ -3,49 +3,49 @@
 <p align="right"><img src="../figures/bogo_sdc.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
-* [6.1 Introduction.](#6.1)
-* [6.2 NFVI APIs.](#6.2)
-  * [6.2.1 Tenant Level APIs.](#6.2.1)
-  * [6.2.2 Hardware Acceleration Interfaces and APIs.](#6.2.2)
-* [6.3 Intra-NFVI Interfaces.](#6.3)
-  * [6.3.1. Hypervisor Hardware Interface.](#6.3.1)
-* [6.4 Enabler Services Interfaces.](#6.4)
+* [6.1 Introduction](#6.1)
+* [6.2 Cloud Infrastructure APIs](#6.2)
+  * [6.2.1 Tenant Level APIs](#6.2.1)
+  * [6.2.2 Hardware Acceleration Interfaces and APIs](#6.2.2)
+* [6.3 Intra-Cloud Infrastructure Interfaces](#6.3)
+  * [6.3.1. Hypervisor Hardware Interface](#6.3.1)
+* [6.4 Enabler Services Interfaces](#6.4)
 
 
 <a name="6.1"></a>
 ## 6.1 Introduction
-In this document’s earlier chapters, the various resources and capabilities of the NFVI have been catalogued and the workloads (VNFs) have been profiled with respect to those capabilities. The intent behind this chapter and an “API Layer” is to similarly provide a single place to catalogue and thereby codify, a common set of open APIs to access (i.e. request, consume, control, etc.) the aforementioned resources, be them directly exposed to the VNFs, or purely internal to the NFVI.
+In this document’s earlier chapters, the various resources and capabilities of the Cloud Infrastructure have been catalogued and the workloads have been profiled with respect to those capabilities. The intent behind this chapter and an “API Layer” is to similarly provide a single place to catalogue and thereby codify, a common set of open APIs to access (i.e. request, consume, control, etc.) the aforementioned resources, be them directly exposed to the workloadss, or purely internal to the Cloud Infrastructure.
```

Comment:
@CsatariGergely For some reason the web UI wouldn't let me edit your commit. There's a small typo, extra "s" at the end of workloadss. Thx, -Mark

Reply:
fixed